src/rtc_peer.ts

```diff
@@ -365,10 +355,15 @@ export class RTCPeer extends EventEmitter {
             // TODO: check whether track is coming from screenshare when we
             // start supporting video.
 
+            let sendEncodings = this.config.simulcast && !isFirefox() ? DefaultSimulcastScreenEncodings : FallbackScreenEncodings;
+            if (opts?.encodings) {
+                sendEncodings = opts.encodings as RTCRtpEncodingParameters[];
+            }
+
             this.logger.logDebug('RTCPeer.addTrack: creating new transceiver on send');
             const trx = this.pc.addTransceiver(track, {
-                direction: 'sendonly',
-                sendEncodings: this.config.simulcast && !isFirefox() ? DefaultSimulcastScreenEncodings : FallbackScreenEncodings,
+                direction: 'sendrecv',
```

Comment:
This helps clever browsers (e.g., Chrome) to optimize the number of transceivers needed. Not a functional change. I checked that screen sharing still works as expected.